pykylin/cursor.py

```diff
@@ -17,15 +24,51 @@ def __init__(self, connection):
         self.fetched_rows = 0
 
     def callproc(self):
-        raise('Stored procedures not supported in Kylin')
+        raise ('Stored procedures not supported in Kylin')
 
     def close(self):
         logger.debug('Cursor close called')
 
+    @staticmethod
+    def trans_sql_for_kylin(sql_str):
```

Comment:
What's the purpose of transforming sql query to another version ?

This is not what a Cursor instance should do. If SQL syntax for Kylin is different from other SQL dialects, you should consider change "dialect.py".

FYI:
[SQLAlchemy Dialect](https://github.com/zzzeek/sqlalchemy/blob/master/README.dialects.rst)